Makefile

```diff
@@ -97,17 +106,23 @@ build-gtest-gmock:
 	(cd testing/googletest && cmake -H. -Bbuild -DCMAKE_C_COMPILER=$(call _arduino_prop,compiler.path)$(call _arduino_prop,compiler.c.cmd) -DCMAKE_CXX_COMPILER=$(call _arduino_prop,compiler.path)$(call _arduino_prop,compiler.cpp.cmd)  .)
 	$(MAKE) -C testing/googletest
 
+.PHONY: adjust-git-timestamps
```

Comment:
I think  this wants to get removed?

Reply:
Yes it does; thanks!  (A mistake born of too many rebases…)